### ISSUE TITLE
/usr/bin/env: ‘node\r’: No such file or directory

### ISSUE THREAD

**Issue by loicnestler** [saltyshiomix/nextron#28]
It looks like nextron was formatted with linux-incompatible new line symbols (`\r\n` instead of `\n`).

Fixed the issue by opening the executable in VSCode by typing `code "$(which nextron)"` and changing the new line symbols from CRLF to LF in the bottom bar.

**Comment by saltyshiomix** [comment]
@loicnestler 

Thank you sooo much for your reporting this bug.
This is same as #2 :

https://github.com/saltyshiomix/nextron/issues/2#issuecomment-403656799

I'll fix this as soon as possible.

Best,

**Comment by saltyshiomix** [comment]
@loicnestler 

I just published `nextron@4.5.6`.
Could you try it out if you have some time?

```bash
# install latest nextron
$ npm install --global nextron@latest

# use it
$ nextron init my-app --template <TEMPLATE-NAME>
$ cd my-app
$ yarn && yarn dev
```

`<TEMPLATE-NAME>` is listed by the command `nextron list`: 

```
$ nextron list

  Available examples:

    - custom-build-options
    - custom-server
    - custom-server-nodemon
    - custom-server-typescript
    - ipc-communication
    - parameterized-routing
    - store-data
    - web-worker
    - with-javascript
    - with-javascript-ant-design
    - with-javascript-emotion
    - with-javascript-material-ui
    - with-python
    - with-typescript
    - with-typescript-emotion
    - with-typescript-less
    - with-typescript-material-ui
    - with-typescript-python-api

  USAGE

    $ nextron init my-app [--template example_folder_name]

  If you want to use "with-typescript-material-ui", just type the command below:

    $ nextron init my-app --template with-typescript-material-ui
```

**Comment by loicnestler** [comment]
@saltyshiomix 
It looks like you fixed the bug. Thank you a lot for your effort!
